### PR TITLE
test(client): add test cases for `.fetchUserInfo()`

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

image

## Testing

the PR itself

## Reviewers

@logto-io/eng

### CODE REVIEW

**Review by simeng-li — APPROVED** [review]

File: Tests/LogtoMock/NetworkSessionMock.swift [hunk 1]
```diff
@@ -27,7 +27,7 @@ public class NetworkSessionMock: NetworkSession {
                         "token_endpoint": "https://logto.dev/token:good",
                         "end_session_endpoint": "https://logto.dev/end:good",
                         "revocation_endpoint": "https://logto.dev/revoke:good",
-                        "userinfo_endpoint": "https://logto.dev/me:good",
+                        "userinfo_endpoint": "https://logto.dev/user",
```
Comment (simeng-li) [inline Tests/LogtoMock/NetworkSessionMock.swift:30]: wondering is anyone referencing it?

**Review by simeng-li — APPROVED** [review]
LGTM

File: Tests/LogtoMock/NetworkSessionMock.swift (hunk 1 above)
Comment (gao-sun) [reply Tests/LogtoMock/NetworkSessionMock.swift:30]: just test cases

Comment (gao-sun) [reply Tests/LogtoMock/NetworkSessionMock.swift:30]: https://github.com/logto-io/swift/pull/52/files#diff-97bcf0f08aaa8e839d22785c49f2e71ce839e8a33eea04ea00c80fca5a27e3d0R47-R57

**Pull request merged**